### PR TITLE
remove chrono

### PR DESCRIPTION
**Pull request by PSeitz**
remove chrono

#1296

### CODE REVIEW

**Review by fulmicoton — APPROVED** [review]

File: Cargo.lock [hunk 1]
```diff
@@ -2702,7 +2701,6 @@ dependencies = [
  "anyhow",
  "async-trait",
  "byte-unit",
- "chrono",
```
Comment (fulmicoton) [inline Cargo.lock:2705]: Is there still a deps on chrono somewhere?
(I was expecting a larger diff in Cargo.lock)

Comment (PSeitz) [reply Cargo.lock:2705]: yes, on diesel. They use it for serialization and don't plan to replace it https://github.com/diesel-rs/diesel/issues/2928

**Review by fmassot — APPROVED** [review]

**Review by fulmicoton — COMMENTED** [review]
Thanks for asking the diesel people. They mentioned it was optional. Do we actually use this feature?

**Comment by PSeitz** [comment]
> Thanks for asking the diesel people. They mentioned it was optional. Do we actually use this feature?

We use it in the postgremodel https://github.com/quickwit-oss/quickwit/blob/main/quickwit-metastore/src/postgresql/model.rs#L64

I think we could replace it and use i64 directly

NativeDateTime seems to be converted to timestamp which is i64

https://github.com/diesel-rs/diesel/blob/50c2c85032857a65e4a8d68dc851fa7ab91d65b2/diesel/src/pg/types/date_and_time/chrono.rs#L45

https://github.com/diesel-rs/diesel/blob/50c2c85032857a65e4a8d68dc851fa7ab91d65b2/diesel/src/pg/types/date_and_time/mod.rs#L22

File: Cargo.lock (hunk 1 above)
Comment (fulmicoton) [reply Cargo.lock:2705]: It seems optional though. Do we need that feature?

**Comment by PSeitz** [comment]
I did a test, and simply replacing the field with i64 will not work with an additional migration step.

**Pull request merged**